go/fn/object.go

```diff
@@ -666,6 +696,14 @@ func (o KubeObjects) Less(i, j int) bool {
 	return idStrI < idStrJ
 }
 
+func DummyKubeObject() *KubeObject {
```

Comment:
This is a public function. IMO this name will be very confusing for users when it's a placeholder for nil KubeObject.

Reply:
Thanks! removed this and use a YNode empty `MapNode` instead.